.github/workflows/publish-github-release.yml

```diff
@@ -0,0 +1,92 @@
+name: Publish GitHub Release as newest
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 3 * * 1-5'
+
+jobs:
+  publish-release:
+    name: Publish GitHub Release
+    runs-on: ubuntu-24.04
+    permissions:
+      contents: write
+      actions: read
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          fetch-depth: 0
+
+      - name: Get latest pre-release name
+        id: get_latest_prerelease
+        run: |
+          latest_release=$(jq -r 'first' <<< $(curl --silent https://api.github.com/repos/bitwarden/android/releases))
```

Comment:
We've been using gh cli which helps simplify these.

* Most recent GitHub Release, including drafts and pre-releases: `gh release list --json name,tagName,isDraft,isPrerelease -L 1`
* Last GitHub Release marked as Released: `gh release view --json name,isPrerelease`

Sending the returned json to `GITHUB_OUTPUT` should enable you to directly access fields in other steps without parsing fields individually. 🤔

Reply:
Thanks. Changed it to gh cli. 

> Sending the returned json to GITHUB_OUTPUT should enable you to directly access fields in other steps without parsing fields individually. 🤔

The data is returned as a list, so I don't think it's possible.

Reply:
You can use `--jq '....'` to retrieve the object:

`gh release list --json isDraft,name,tagName -L 1 --jq '.[]'` something like that should work.